go.mod

```diff
@@ -79,6 +79,8 @@ require (
 	sigs.k8s.io/yaml v1.2.0
 )
 
+require github.com/DATA-DOG/go-sqlmock v1.5.0
```

Comment:
This relates to https://github.com/kyma-incubator/reconciler/issues/1008 and could be a partial solve for our DB Unit Tests FYI @adityabhatia @Tomasz-Smelcerz-SAP